lib/mix/tasks/phx.gen.html.ex

```diff
@@ -0,0 +1,138 @@
+Code.require_file("../../../installer/lib/phx_new/generator.ex", __DIR__)
```

Comment:
We should not use require_file on compiled code. The installer directory is likely not going to be there once we push the package.